sdk/servicebus/service-bus/src/modelsToBeSharedWithEventHubs.ts

```diff
@@ -3,32 +3,23 @@
 
 // TODO: this code is a straight-copy from EventHubs. Need to merge.
 
-import { AbortSignalLike } from "@azure/abort-controller";
 import { Span, SpanContext } from "@opentelemetry/api";
-import { OperationTracingOptions } from "@azure/core-tracing";
+import { OperationOptions } from "@azure/core-http";
 
 /**
  * Options for configuring tracing and the abortSignal.
  */
 // NOTE: This class is intended to mirror the relevant fields and structure from
 // @azure/core-http OperationOptions
-export interface OperationOptions {
-  /**
-   * The signal which can be used to abort requests.
-   */
-  abortSignal?: AbortSignalLike;
-  /**
-   * Options for configuring tracing.
-   */
-  tracingOptions?: OperationTracingOptions;
-}
+export type OperationOptionsBase = Pick<OperationOptions, "abortSignal" | "tracingOptions">;
+export { OperationOptions };
```

Comment:
Why should `OperationOptions` be exported from here? Can we not update `index.ts` to export it directly from `@azure/core-http` instead?

Though it doesnt make any net difference to the end user, it is misleading to folks working on the project to see `import {OperationOptions} from "./modelsToBeSharedWithEventHubs"` and wonder why should this be shared with Event Hubs

Reply:
Updated